src/include/OpenImageIO/thread.h

```diff
@@ -715,7 +715,10 @@ class OIIO_API thread_pool {
     /// Return true if the calling thread is part of the thread pool. This
     /// can be used to limit a pool thread from inadvisedly adding its own
     /// subtasks to clog up the pool.
-    bool this_thread_is_in_pool() const;
+    /// DEPRECATED(2.1) -- use is_worker() instead.
```

Comment:
Oh, I just realized, I need to make this non-inline in order to not break compatibility when I backport to the release branch.